src/rx.rs

```diff
@@ -239,6 +234,7 @@ impl<'a> RxRing<'a> {
         // Register RxDescriptor
         eth_dma
             .dmardlar
+            // Note: unsafe block required for `stm32f107`.
             .write(|w| unsafe { w.srl().bits(ring_ptr as u32) });
```

Comment:
is there clippy warning for others CPUs ? maybe it should be added `allow(unused_unsafe)`

Reply:
No warnings for any other CPU - I'm removing the note